src/applications/ivc-champva/10-7959f-1/containers/IntroductionPage.jsx

```diff
@@ -5,13 +5,22 @@ import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 import { VaAlert } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { Link } from 'react-router';
+import { getNextPagePath } from '@department-of-veterans-affairs/platform-forms-system/routing';
+import recordEvent from 'platform/monitoring/record-event';
 
 const IntroductionPage = props => {
- 10-7959f-1-increase-tests
-  const { route } = props;
-  const { formConfig, pageList } = route;
+  const { route, isLoggedIn } = props;
+  const { formConfig, pageList, formData, pathname } = route;
 
-  const firstPage = pageList[1]?.path;
+  const getStartPage = () => {
+    const data = formData || {};
+    if (pathname) return getNextPagePath(pageList, data, pathname);
+    return pageList[1].path;
+  };
+
+  const handleClick = () => {
+    recordEvent({ event: 'no-login-start-form' });
+  };
```

Comment:
This (and the other changes) were added in to show the different alerts based on user logged in status and route to the correct page when starting the form